.github/workflows/cross-version-tests.yml

```diff
@@ -75,14 +75,13 @@ jobs:
       - id: set-matrix
         name: Set matrix
         run: |
-          if [ ! "$(git branch --show-current)" == "master" ]; then
-            git fetch origin master:master
-          fi
-
           EVENT_NAME="${{ github.event_name }}"
           if [ "$EVENT_NAME" = "pull_request" ]; then
-            REF_VERSIONS_YAML="https://raw.githubusercontent.com/mlflow/mlflow/master/mlflow/ml-package-versions.yml"
-            CHANGED_FILES="$(git diff --name-only master..HEAD)"
+            REPO="${{ github.repository }}"
```

Comment:
Would `REPO` always be `mlflow/mlflow`? Why do we want to replace this? I guess this will not be the user's forked repo (e.g. `liangz1/mlflow`), correct?

Reply:
> Would REPO always be mlflow/mlflow?

No, you can open a PR on your fork (I sometimes do).

Reply:
What if my fork repo's remote master is many many commits behind the mlflow master? I never push back to my fork's remote master since that's not necessary. If so, I guess my PRs change list will be monotonically growing. Would this be a problem?

Reply:
That should not be a problem. When you open a PR on `mlflow/mlflow`, GitHub compares your changes against the master of `mlflow/mlflow`.